src/gosling-track/gosling-track.ts

```diff
@@ -133,54 +134,51 @@ const factory: PluginTrackFactory<Tile, GoslingTrackOptions> = (HGC, context, op
     /* Custom loading label */
     const loadingTextStyle = getTextStyle({ color: 'black', size: 12 });
 
+    /**
+     * The main plugin track in Gosling. This is a versetile plugin track for HiGlass which relies on GoslingTrackModel
+     * to keep track of mouse event and channel scales.
+     */
     class GoslingTrackClass extends BarTrack<Tile, typeof options> {
-        // This is tracking the xScale of an entire view, which is used when no tiling concepts are used
-        drawnAtScale = HGC.libraries.d3Scale.scaleLinear();
-        scalableGraphics: Record<string, PIXI.Graphics> = {};
-        extent = { min: Number.MAX_SAFE_INTEGER, max: Number.MIN_SAFE_INTEGER };
-        hideMousePosition?: () => void;
-
-        private tileSize: number;
-        private mRangeBrush: LinearBrushModel;
-
-        private assembly?: Assembly;
-        private processedTileInfo: Record<string, ProcessedTileInfo>;
-        // TODO: add members that are used explicitly in the code
-        //
-        // Brushes on the color legend
-        private gLegend? = HGC.libraries.d3Selection.select(context.svgElement).append('g');
-        private pMouseHover = new HGC.libraries.PIXI.Graphics();
-        private pMouseSelection = new HGC.libraries.PIXI.Graphics();
-        private isRangeBrushActivated = false;
-        private gBrush = HGC.libraries.d3Selection.select(context.svgElement).append('g');
-
-        private loadingTextStyleObj = new HGC.libraries.PIXI.TextStyle(loadingTextStyle);
-        private loadingTextBg = new HGC.libraries.PIXI.Graphics();
-        private loadingText = new HGC.libraries.PIXI.Text('', loadingTextStyle);
-
-        private svgData: unknown[] = [];
-        private textGraphics: unknown[] = [];
-        /** used to improve the performance of text rendering */
-        private textsBeingUsed = 0;
-        private loadingStatus = { loading: 0, processing: 0, rendering: 0 };
-
-        private prevVisibleAndFetchedTiles?: Tile[];
-        private oldDimensions?: [number, number];
-        private xDomain?: number[];
-        private xRange?: number[];
-
-        private mouseDownX = 0;
-        private mouseDownY = 0;
-
-        /** Store the color legends added so far so that we can avoid overlaps and redundancy */
-        private displayedLegends: DisplayedLegend[] = [];
+        /* *
+         *
+         *  Properties
+         *
+         * */
+
+        tileSize: number;
```

Comment:
This has to be public or things break...but I don't see it being used in HiGlass